vizro-core/tests/integration/test_examples.py

```diff
@@ -1,5 +1,6 @@
 # ruff: noqa: F403, F405
```

Comment:
This file is the major change. Now we can have a single parametrised test rather than 4 separate tests.